about.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link rel="stylesheet" href="styles/about.css">
+    <!-- Bootstrap CSS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+
+    <!-- Fontawesome CDN -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      crossorigin="anonymous"
+    />
+
+    <!-- Font laro family -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+
+<link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+
+</head>
+
+<body id="body">
+
+  <header>
+      <!--  Desktop-Navbar -->
+        <!-- Upper Navbar -->
+      <nav class="navbar navbar-expand-md bg-dark upper-navbar d-none d-md-block">
+        <div class="container d-flex justify-content-end very-up-nav">
+          <div class="d-flex">
+            <ul class="d-flex">
+              <li class="me-3 ">
+                <i class="fab fa-facebook-f header-social-links"></i>
+              </li>
+              <li>
+                <i class="fab fa-twitter header-social-links"></i>
+              </li>
+            </ul>
+            <ul class="d-flex">
+              <li class="me-3 top-header-links">English</li>
+              <li class="me-3 top-header-links">My Page</li>
+              <li class="top-header-links">Logout</li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+
+        <!-- Lower Navbar -->
+        
+      <nav class="navbar navbar-expand-md bg-light hov border border-bottom border-white lower-nav d-none d-md-block">
+        <div class="container">
+          <div class="navbar-collapse mt-2">
+            <a href="index.html" class="navbar-brand" id="logo">Concert-Hub</a>
+            <ul class="navbar-nav ms-auto" >
+              <li class="nav-item" >
+                <a href="index.html" class="nav-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Events</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+            </ul>
+          </div> 
+        </div>
+      </nav>
+  </header>
```

Comment:
- [ ] On mobile view on the about page, we can't navigate from that about page to the home page because when we click on the hamburger icon, the menu is not being displayed as it should. Please try and fix that.